packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -170,10 +145,8 @@ contract MerkleDistributor is Ownable {
      *         will revert. It also reverts when `_claim`'s `amount` exceeds `remainingAmount` for the window.
      * @param _claim claim object describing amount, accountIndex, account, window index, and merkle proof.
      */
-    function claim(Claim memory _claim) external {
-        _verifyAndMarkClaimed(_claim);
-        merkleWindows[_claim.windowIndex].remainingAmount -= _claim.amount;
-        merkleWindows[_claim.windowIndex].rewardToken.safeTransfer(_claim.account, _claim.amount);
+    function claim(Claim memory _claim) external virtual override {
+        _executeClaim(_claim);
```

Comment:
why does this need to be an internal function like this? it's only uses here and nowhere else, right?

Reply:
Good point will remove